docs/diagrams/CustomerAddActivityDiagram.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
+skin rose
+skinparam ActivityFontSize 15
+skinparam ArrowFontSize 12
+!pragma useVerticalIf on
+start
+:User executes customer add command;
+:Parse command string;
+
+if () then ([else])
+    :ParseException: Invalid Command Format;
+    stop
+([all fields are filled and valid]) elseif () then ([else])
```

Comment:
Could consider adding a space between `(` and `[` such that the square
bracket does not overlap with the lines in the activity diagram.